modules/table/src/test/java/org/apache/ignite/internal/table/distributed/raft/snapshot/outgoing/SnapshotAwarePartitionDataStorageTest.java

```diff
@@ -166,11 +174,11 @@ void delegatesAcquirePartitionSnapshotsReadLock() {
     }
 
     @ParameterizedTest
-    @EnumSource(WriteAction.class)
-    void writingNotYetPassedRowIdForFirstTimeSendsEnqueuesItOnSnapshotOutOfOrder(WriteAction writeAction) {
+    @EnumSource(MvWriteAction.class)
+    void writingNotYetPassedRowIdForFirstTimeEnqueuesItOnSnapshotOutOfOrder(MvWriteAction writeAction) {
```

Comment:
Oh my god! Can you move this to the comment? I don't see a sentence here, all I see is a bunch of characters that form a big string. This is too much. I guess I didn't read these titles during the last review

Reply:
Shortened the names